psh/test-kill.c

```diff
@@ -0,0 +1,54 @@
+/*
+ * Phoenix-RTOS
+ *
+ * phoenix-rtos-tests
+ *
+ * executable for test-kill.py
+ *
+ * Copyright 2021 Phoenix Systems
+ * Author: Damian Loewnau
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <errno.h>
+#include <libgen.h>
+#include <unistd.h>
+
+int main(int argc, char *argv[])
+{
+	int pid, i;
+	char *const arg[][9] = {
+		{ "test_proc_0", NULL },
+		{ "test_proc_1", NULL },
+		{ "test_proc_2", NULL }
+	};
+	char *const bsname = basename(argv[0]);
+
+	if (strstr(bsname, "test_proc_")) {
+		while (1)
+			;
```

Comment:
Maybe usleep/sleep here? This will cause 100% CPU usage

Reply:
Thanks, to make it simper, I leave only psh processes testing. I will appreciate, if you have a look second time, to push it.

Reply:
Finally I removed additional c executable